db/models.py

```diff
@@ -50,3 +56,74 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User",
+                             on_delete=models.CASCADE,
+                             related_name="orders")
```

Comment:
Code Style: It's better to use `get_user_model()` instead of directly referencing the `User` model. This is a best practice in Django to avoid potential issues with custom user models.